features/logging/4.x/cluster_logging_operator.feature

```diff
@@ -0,0 +1,33 @@
+@clusterlogging
+Feature: cluster-logging-operator related test
+
+  # @author qitang@redhat.com
+  # @case_id OCP-21333
+  # @case_id OCP-19875
+  # @case_id OCP-24111
+  # @case_id OCP-24578
```

Comment:
Please check [DOC-1125811](https://mojo.redhat.com/docs/DOC-1125811) for info about the `case_id` tag.

-- saved reply

Reply:
otherwise looks good as long as your team is happy with it.

Reply:
@akostadinov I missed some checkpoints here, I'll divide this scenario into 2 scenarios. Thanks